.travis.yml

```diff
@@ -19,6 +19,7 @@ install: true
 script:
   - |
     if [ $MODE == current ]; then
+      ./samples/jvm/spek/gradlew -p ./samples/jvm/spek build       
```

Comment:
Please in an own if as I wrote. this way we would not see that it fails

Reply:
didn't get what you mean?

Reply:
Only the last statement within an if counts towards the exit code of the whole if. So if the first statement fails, then the second is still carried out and if this one succeeds, then the whole if succeeds. Therefore, don't but the additional `gr` inside the existing if but duplicate the if.